test/unit/processing/confirm-due-dates.test.js

```diff
@@ -110,37 +114,37 @@ describe('confirm due dates', () => {
   test('should not update schedule when monthly and no outstanding payments', () => {
     const paymentRequests = [{
       ledger: AP,
-      dueDate: '09/11/2021',
-      schedule: 'M4',
+      dueDate: '09/11/2020',
+      schedule: 'M12',
       value: -100
     }]
     const previousPaymentRequests = [{
       paymentRequestNumber: 1,
       ledger: AP,
-      dueDate: '08/09/2021',
-      schedule: 'M4',
+      dueDate: '08/09/2020',
+      schedule: 'M12',
       value: 1000,
       settledValue: 1000
     }]
     const confirmedPaymentRequests = confirmDueDates(paymentRequests, previousPaymentRequests, currentDate)
-    expect(confirmedPaymentRequests[0].dueDate).toBe('09/11/2021')
-    expect(confirmedPaymentRequests[0].schedule).toBe('M4')
+    expect(confirmedPaymentRequests[0].dueDate).toBe('09/11/2020')
+    expect(confirmedPaymentRequests[0].schedule).toBe('M12')
   })
 
   test('should update schedule to cover remaining payments when monthly and three remaining', () => {
```

Comment:
Test case when monthly and 11 remaining (extreme edge case) -- can do all inclusive as you have Q4 and T4 but not required

Reply:
Fair comment.  Originally that wasn't needed as we assumed the frequency was derived.  Now we know it's not and we know all permutations makes sense to cover them with tests.